templates/iso/8.3/.treeinfo

```diff
@@ -1,3 +1,11 @@
+[platform]
+name = @@PLATFORM_NAME@@
+version = @@PLATFORM_VERSION@@
+
+[branding]
+name = @@PRODUCT_BRAND@@
+version = @@PRODUCT_VERSION@@
+
```

Comment:
Those should at least be flagged with as "legacy" using a comment.

Reply:
Are they legacy if an upgrade path requires them?

Reply:
Ping.